openpype/hosts/maya/api/lib.py

```diff
@@ -130,7 +131,13 @@ def float_round(num, places=0, direction=ceil):
 def pairwise(iterable):
     """s -> (s0,s1), (s2,s3), (s4, s5), ..."""
     a = iter(iterable)
-    return itertools.izip(a, a)
+
+    if sys.version_info[0] == 2:
+        izip = itertools.izip
+    else:
+        izip = zip
+
+    return izip(a, a)
```

Comment:
Maybe instead of checking Python version inside the function, just add `from six.moves import zip` statement above would be better. :)

Reply:
Implemented with https://github.com/pypeclub/OpenPype/pull/2445/commits/d3eca27140e7068dc9a05f1336992da8b9060f56